starter-code/main.js

```diff
@@ -53,7 +53,188 @@ mongoClient.connect(`mongodb://localhost:27017/crunchbase`, (error, db) => {
             })
             break;
 
-          // Code here next cases!
+          case "4":
+            // 4.- List by name all companies founded in february of 2004.
+            db.collection('companies').find({ $and: [{ "founded_year": 2004 }, { "founded_month": 2 }] }, { name: 1, _id: 0 }).toArray((error, result) => {
```

Comment:
Aquí no son necesarias las comillas: 

`.find({ $and: [{ founded_year: 2004 }, { founded_month: 2 }] }, { name: 1, _id: 0 })`